tests/test_save_output.py

```diff
@@ -96,19 +84,11 @@ def test_save_output():
     # load the output and plot the sources
     scene_loaded = model_from_h5(filename, ID, path=path)
     print("Output loaded from h5 file")
-    # plotting
-    norm = plot.AsinhAutomaticNorm(obs)
-    plot.scene(
-        scene_loaded,
-        obs,
-        norm=norm,
-        show_model=True,
-        show_rendered=True,
-        show_observed=True,
-        show_residual=True,
-    )
-    plt.show()
 
+    # compare scenes 
+    saved = jax.tree_util.tree_structure(scene_)
+    loaded = jax.tree_util.tree_structure(scene_loaded)
+    print(f"saved == loaded: {saved == loaded}")
```

Comment:
Can you add the test for `jax.tree_util.tree_leaves`, please. It probably complains about missing `__eq__`, in which case I'd add something like `for leave1, leave2 in zip(leaves1, leaves2): assert jnp.allclose(leave1, leave2)`.

Reply:
check updated now